src/ORM/Versioning/ChangeSet.php

```diff
@@ -535,6 +541,20 @@ public function getDescription()
         return $string;
     }
 
+    /**
+     * Get number of changes
+     *
+     * @return int
+     */
+    public function getChangesCount()
```

Comment:
The method is needed anyway because, I couldn't use `Changes.Count` due to the fact that React gridfield component doesn't yet support composite summary fields.